docs/guides/auth.md

```diff
@@ -47,7 +47,7 @@ provider "hcp" {
   client_secret = "service-principal-key-client-secret"
 }
 ```
--> **Note:** If a [Project-Level Service Principal](https://cloud.hashicorp.com/docs/hcp/admin/iam/service-principals) is used, specify the default `project_id` in your provider configuration.
+-> **Note:** If a [Project-Level Service Principal](https://cloud.hashicorp.com/docs/hcp/admin/iam/service-principals) is used, specify the default `project_id` in your provider configuration or set the HCP_PROJECT_ID environment variable.
```

Comment:
I don't think this env variable is strictly for project level SPs. We can add something like this 
```
When multiple projects exist within the organization, a project ID should be set in one of the following ways: 

- Using the HCP_PROJECT_ID  environment variable

-  On the provider:
provider "hcp" {
 project_id = "Your project ID"
}


- on each resource:
resource "hcp_hvn" "test" {
  hvn_id         = "test-hvn"
  project_id = <your project ID>
  cloud_provider = "aws"
  region         = "us-west-2"
}


If project ID is not set, the oldest project in the organization will be used by default.
```